Remove the defaults from the style wrapper injection. Remove the mean… (#3794)
plone/volto: docs/source/upgrade-guide/index.md

It moved to use `react-virtualized` instead of `react-window` because it provides a more broad set of features that Volto required.
If you were using it in your project, you'll have to include it as a direct dependency of it from now on.

> ### Change the way the Style Wrapper is enabled and how to add the `styles` field

Reviewer: ```suggestion
### Change the way the style wrapper is enabled and how to add the `styles` field
```

> During the feedback we had during the alpha stage, we have determined that it's difficult to deal with a separate way to define (and extend) the styles schema.

Reviewer: ```suggestion
During the alpha stage, we received feedback and determined that it's too difficult to deal with a separate way to define (and extend) the styles schema.
```

> We decided that the best is to deal with it as any other schema fields and enhance it via schema enhancers.

Reviewer: ```suggestion
We decided it is best to deal with it as any other schema field and enhance it via schema enhancers.
```

> This improves the developer experience specially when dealing with variations that can provide their own styles and other schema fields.

Reviewer: ```suggestion
This improves the developer experience, especially when dealing with variations that can provide their own styles and other schema fields.
```
